main.nf

```diff
@@ -326,19 +356,19 @@ process multiqc {
     publishDir "${params.outdir}/multiqc"
 
     input:
-    file ('supernova/') from supernova_results2.flatten().toList()
-    file ('busco/') from busco_results.flatten().toList()
-    file ('quast/') from quast_results.flatten().toList()
-    file ('software_versions/') from software_versions_yaml.flatten().toList()
+    file ('supernova/') from supernova_results2.toList()
+    file ('busco/') from busco_results.toList()
+    file ('quast/') from quast_results.toList()
+    file ('software_versions/') from software_versions_yaml.toList()
```

Comment:
I don't even remember if I had any reason for writing `.flatten().toList()` in the first place.